### PR TITLE
Print stacktrace for console appender

### PR DESCRIPTION
**Pull request by doncem**
Custom layout did not print out stacktrace for console appender. Adding and testing

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java [hunk 1]
```diff
@@ -47,9 +46,7 @@ private static void triggerBadImplementationLog(Throwable cause) {
         log.error("Bad implementation of '" + cause.getClass().getCanonicalName() + "' in use", invalid);
     }
 
-    public static AbstractLoggingException getFromLogEvent(ILoggingEvent event) {
-        ThrowableProxy proxy = (ThrowableProxy) event.getThrowableProxy();
-
+    public static AbstractLoggingException getFromLogEvent(ThrowableProxy proxy) {
```
Comment (rlewan) [inline src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java]: I think we should change the method name as well, as it no longer operates on a log event.

File: src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java [hunk 2]
```diff
@@ -1,6 +1,5 @@
 package uk.gov.hmcts.reform.logging.exception;
 
-import ch.qos.logback.classic.spi.ILoggingEvent;
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java]: do you want to bump the version here, or a separate change?

File: src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java (hunk 1 above)
Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java]: oopsie

File: src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java (hunk 2 above)
Comment (rlewan) [reply src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java]: Would be nice to bump here and have it all done.

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java]: you meant overall pr, not this single line change, right?

**Pull request merged**